Improvements for students, instructors and contributors

shawnrhoads opened the pull request:
This PR improves content and documentation per [JOSE reviews](https://github.com/openjournals/jose-reviews/issues/146#issuecomment-985716142).

Related to #180
- Separates documentation for students, instructors, and contributors in `README.md` and `docs/index.md`
- Added three issue forms for bugs, enhancements, and questions

Related to #182
- Removed the quadratic equation example from Module 02-02

gllg4009 commented on .github/ISSUE_TEMPLATE/enhancement-template.yml:
```diff
@@ -0,0 +1,55 @@
+    attributes:
+      label: Other information
+      description:
+        Anything else that might be helpful? E.g., related issues/enhancement, suggestions how to improve, links/references with relevant examples
```
"related issues/**enhancements**"

"suggestions **on** how to improve"

gllg4009 commented on .github/ISSUE_TEMPLATE/enhancement-template.yml:
```diff
@@ -0,0 +1,55 @@
+name: Enhancement Report
```
why would this table be on top of the form?

shawnrhoads replied:
The table is only visible in the preview, not the actual form.

gllg4009 commented on .github/ISSUE_TEMPLATE/issue-template.yml:
```diff
@@ -0,0 +1,66 @@
+    id: description
+    attributes:
+      label: Description
+      description: What is the issue (e.g., typo, bug, inaccurate descriptions, etc.)? Please tell us what you see. What did you expect?
```
"**typos, bugs**"

gllg4009 commented on .github/ISSUE_TEMPLATE/issue-template.yml:
```diff
@@ -0,0 +1,66 @@
+    attributes:
+      label: If applicable, what browsers are you seeing the problem on?
+      multiple: true
+      options:
```
add "other"?

shawnrhoads replied:
instead of adding "other", I'll change it to "what browser(s)"

gllg4009 commented on README.md:
```diff
@@ -1,65 +1,71 @@
-Please visit <a target="_blank" rel="noopener noreferrer" href="https://shawnrhoads.github.io/gu-psyc-347/module-00-06_Contributing.html">this page</a> if you would like to help improve and/or expand the content in this Jupyter Book!
+## For Contributors 🤝
+
+While the course aims to incorporate a diverse curriculum to introduce students to the basics of modeling in the context of social behavior, there is still so much still left out (and still much unknown in this growing field). That being said, other researchers and educators are invited to help improve and expand the content included in this Jupyter Book!
```
"there is still so much still left out (and still much unknown in this growing field)": a lot of "still"s!

shawnrhoads replied:
good catch!

gllg4009 commented on CONTRIBUTING.md:
```diff
@@ -16,15 +16,47 @@ The goals of this course (as outlined in the [syllabus](https://shawnrhoads.gith
-- If you spot an error (e.g., typo, bug, inaccurate descriptions, etc.), please open a new issue on GitHub by clicking on the GitHub Icon in the top right corner on any page and selecting "open issue". Alternatively, you can [open a new issue directly through GitHub](https://github.com/shawnrhoads/gu-psyc-347/issues/new?title=Issue%20on%20page:%20*html%20&body=Your%20issue%20description%20here).
-- If I had inadvertently omitted credit for any content generated by others, please also let me know.
-- If you have an idea for a tutorial or a new module to include (especially related to the [growing list of readings](https://shawnrhoads.github.io/gu-psyc-347/module-00-03_Reading-List.html)), please either [open a new issue](https://github.com/shawnrhoads/gu-psyc-347/issues/new?title=Tutorial%20Idea:%20[INSERT%20TITLE]&body=Please%20suggest%20a%20tutorial%20idea%20here%20with%20any%20relevant%2 ...
+- If you spot an error (e.g., typo, bug, inaccurate descriptions, etc.), please open a new issue on GitHub by clicking on the GitHub Icon in the top right corner on any page and selecting "open issue". Alternatively, you can [open a new issue directly through GitHub](https://github.com/shawnrhoads/gu-psyc-347/issues/new?labels=bug&template=issue-template.yml).
```
how does one access the different issue templates?

shawnrhoads replied:
the links should work once this PR is merged into master

gllg4009 commented on README.md:
```diff
@@ -1,65 +1,71 @@
+
+All course materials are also designed to be modular. For example, if students are knowledgeable in Python basics and Jupyter Notebook (**Module 01**), then instructors could begin with model fitting (**Module 02**). If students are knowledgeable in model fitting with maximum likelihood estimation (**Module 02**), instructors could start with reinforcement learning (**Module 03**). Instructors ca ...
+
+For additional background on the content covered in each tutorial (technical or theoretical), instructors are encouraged to review the references and/or resources listed at the beginning of each tutorial. In addition, instructors should familarize themselves with the <a target="_blank" rel="noopener noreferrer" href="https://shawnrhoads.github.io/gu-psyc-347/module-00-03_Reading-List.html">recomm ...
```
"review the references and/or resources listed at the beginning of each tutorial": most tutorials didn't seem to have many resources

shawnrhoads replied:
Good point! I added a new `RESOURCES.md` page. I also updated some info in the instructors section.

gllg4009 commented on docs/module-02-01_Nonlinear-Modeling.ipynb:
```diff
@@ -51,10 +51,6 @@
     "\n",
     "Nonlinear modeling simply implies that the relationship between $y$ and $x_d$ is more than a linear combination. Some common examples of nonlinear models include\n",
     "\n",
-    "**Quadratic function**: \n",
```
the colab file accessed through the link on top of the Jupiter notebook doesn't incorporate these new changes

shawnrhoads replied:
Oh right.. this will also change once this PR is merged into `master`. 

See this link which opens the notebook from the `jose-edits` branch: https://colab.research.google.com/github/shawnrhoads/gu-psyc-347/blob/jose-edits/docs/module-02-01_Nonlinear-Modeling.ipynb

gllg4009 reviewed (commented):
Hi Shawn! Please see comments

shawnrhoads commented:
Hi @gllg4009! Thanks for reviewing!

I made the following changes for your re-review:
- fixed typos
- added a `RESOURCES.md` page (repo only) - can you check for typos and broken links?
- updated text for instructors - can you check for typos, clarity, and broken links?

Let me know if you have any questions or if anything else needs editing/clarifying.

gllg4009 commented on RESOURCES.md:
```diff
@@ -0,0 +1,83 @@
+- [Collection of Jupyter Notebooks](https://github.com/jupyter/jupyter/wiki)
+- [The homogenization of scientific computing, or why Python is steadily eating other languages’ lunch](https://www.talyarkoni.org/blog/2013/11/18/the-homogenization-of-scientific-computing-or-why-python-is-steadily-eating-other-languages-lunch/)
+- [Shawn Rhoads - A brief introduction to Python for psychological science research](https://www.apa.org/science/about/psa/2019/07/python-research)
+- [R or Python for Psychologists](https://dominiquemakowski.github.io/post/r_or_python/)
```
broken link

gllg4009 reviewed (commented):
see comment above

shawnrhoads commented:
Thanks, @gllg4009! Once you approve these, we can merge to `master`!
